src/utils/validations.js

```diff
@@ -123,6 +123,11 @@ export const validateVmLikeEntityName = (value, vmSettings, props) => {
       );
 };
 
+export const validateCloudInitHostName = value => {
+  const dnsValidation = validateDNS1123SubdomainValue(value);
+  return dnsValidation && dnsValidation.type === VALIDATION_ERROR_TYPE ? dnsValidation : null;
```

Comment:
I don't think we have to check for the error_type here, because all the errors will be error type or none in this case. Nevertheless I do think we should check for empty error here, because the host name can be empty if I am not mistaken.

Reply:
You're correct. Done!